### PR TITLE
TAG-1980 Adding claim endpoint

### PR DESCRIPTION
**Pull request by cesarhernandezgt**
This updates the signature and basic auth project.

### CODE REVIEW

**Comment by brunobat** [comment]
+1

File: basic-auth/pom.xml [hunk 1]
```diff
@@ -33,6 +33,12 @@
       <scope>provided</scope>
     </dependency>
 
+
+    <dependency>
+      <groupId>com.tomitribe.tribestream.examples</groupId>
+      <artifactId>claims-data</artifactId>
+      <version>0.81-SNAPSHOT</version>
```
Comment (jgallimore) [inline basic-auth/pom.xml]: Is that the right version number? Could it be ${project.version}?

File: basic-auth/src/main/java/org/supertribe/basicauth/ClaimSinglenton.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import com.tomitribe.tribestream.examples.Claim;
+import com.tomitribe.tribestream.examples.ClaimsLoad;
+import javax.ejb.Singleton;
+import java.util.HashMap;
+
+/**
+ * Singleton bean used for loading the hashmap of claims used by the Colors endpoint.
+ */
+@Singleton
+public class ClaimSinglenton {
```
Comment (jgallimore) [inline basic-auth/src/main/java/org/supertribe/basicauth/ClaimSinglenton.java]: Slight typo in this class name: ClaimSingleton. You could drop 'Singleton' from the name altogether - ClaimStore or something like that could work.

**Comment by cesarhernandezgt** [comment]
@jgallimore @brunobat  I fixed the typo and renamed the file to ClaimStore.java along with it usages.

**Pull request merged**